test/e2e/common/pods.go

```diff
@@ -226,48 +209,9 @@ var _ = framework.KubeDescribe("Pods", func() {
 		})
 	})
 
-	It("should be schedule with cpu and memory limits [Conformance]", func() {
```

Comment:
Mentioned in the 1st comment of the PR:
It doesn't make sense at the node level.
It should have been tested in scheduler_predicates at the cluster level https://github.com/kubernetes/kubernetes/blob/master/test/e2e/scheduler_predicates.go#L264

:)